### PR TITLE
Add issue template

### PR DESCRIPTION
**Pull request by girtis**
Adds a template for GitHub issues.

### CODE REVIEW

**Review by bobiechen-twilio — CHANGES_REQUESTED** [review]
LGTM except the minor s/Video/Voice comment

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+// Log output when the issue occurs
+```
+
+### Versions
+
+All relevant version information for the issue.
+
+#### Video iOS SDK
+
+[e.g. 1.2.0 via CocoaPods]
```
Comment (ceaglest) [inline .github/ISSUE_TEMPLATE.md]: s/Video/Voice. Maybe use `2.0.0-beta13` as the version example?

**Review by ceaglest — CHANGES_REQUESTED** [review]
Awesome thanks for the PR! Once the version section is updated this is good to go.

**Review by ceaglest — APPROVED** [review]

**Review by bobiechen-twilio — APPROVED** [review]
👍 Thanks @girtis

**Pull request merged**